test/specs/sortByPriceTest.regression.js

```diff
@@ -1,10 +1,14 @@
 import HomePage from '../pageobjects/home.page.js'
 import LoginPage from '../pageobjects/login.page.js'
 
+
 const executeSortTest = async (sortOption) => {
+
+  console.log('Username:', username)
```

Comment:
In files where we reference global vars, it is good practice to make the first line of the file /* global var1, var2, var3 ... */